src/util/modifications.cpp

```diff
@@ -172,7 +172,6 @@ AA_MOD_T::AA_MOD_T(int mod_idx) {
   symbol_ = mod_sqt_symbols[mod_idx];
   identifier_ = mod_id_masks[mod_idx];
   prevents_cleavage_ = false;
-  prevents_xlink_ = false;
   mono_link_ = false;
```

Comment:
Shouldn't this be removed as well? Or this is that Kojak parameter you mentioned? It looks like getter and setter for this member are not used anywhere.